packages/frontity-org-theme/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import FrontityOrg from "../types";
 import Theme from "./components";
-import { backgroundColor, textColor } from "./processors";
+import { boxShadow , backgroundColor, textColor } from "../src/processors";
```

Comment:
Why do we change here the folder? Shouldn't it be `"./processors"`?

Reply:
I guess all of the code (except some configs) that powers that applications should be in the source `src` folder.

Reply:
> I guess all of the code (except some configs) that powers that applications should be in the source src folder.

You are right about that, but that's already the case. The imports `"./processors"` and `"../src/processors"` are equivalent.

Please note that with `..` you are going up a folder, so you need to indicate `/src` to come back to the same folder where you are. 
On the other hand, `.` just means "the same folder where you are", so you end up in the same place in both cases.

If this is not clear let me know and I'll give you a more detailed explanation.